project.clj

```diff
@@ -3,10 +3,13 @@
   :url "https://github.com/weavejester/medley"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
+  ;; NOTE: :dependencies not included for ClojureCLR as per lein-clr version
+  ;; 0.2.2
```

Comment:
I can remove the comment. From what I've seen most people still use lein-clr 0.2.1 so that they can include Clojars dependencies for cljr, which can obviously be useful, but for this project no cljr dependencies are needed so this newer version is preferable so that the clj dependency (e.g. [org.clojure/clojure "1.9.0"]) that is incompatible with cljr doesn't cause any conflicts.